dotnet/src/SemanticKernel.Core/Text/TextChunker.cs

```diff
@@ -25,8 +25,8 @@ public static class TextChunker
     public delegate int TokenCounter(string input);
 
     private static readonly char[] s_spaceChar = new[] { ' ' };
-    private static readonly string?[] s_plaintextSplitOptions = new[] { "\n\r", ".", "?!", ";", ":", ",", ")]}", " ", "-", null };
-    private static readonly string?[] s_markdownSplitOptions = new[] { ".", "?!", ";", ":", ",", ")]}", " ", "-", "\n\r", null };
+    private static readonly string?[] s_plaintextSplitOptions = new[] { "\n\r", ".", "。", "．", "?!", ";", ":", ",", "，", "、", ")]}", " ", "-", null };
```

Comment:
I didn't add the full-width exclamation and question marks, or the like of ideographic brackets because it's pretty long https://www.w3.org/TR/jlreq/#cl-02